index.html

```diff
@@ -270,87 +329,289 @@
 It will continue to evolve and the task force will issue updates as often as needed. At the
 conclusion of the task force, the TAG intends to adopt this document as a Finding.
 
+<aside class="note">
+
+Privacy is covered by legal frameworks and this document recognises that existing data
+protection laws take precedence for legal matters. However, because the Web is global, we
+benefit from having shared concepts to guide its evolution as a system built for the people using it
+([[?RFC8890]]). A clear and well-defined view of privacy on the Web, informed by research,
+can hopefully help all the Web's participants in different legal regimes. Our shared
+understanding is that the law is a floor, not a ceiling.
+
+</aside>
+
 </section>
 
-# Introduction {#intro}
+# Understanding Privacy Principles {#intro}
+
+Information is power. It can be used to predict and to influence people, as well as to design online
+spaces that control people's behaviour. The collection and [=processing=] of information in greater
+volume, with greater precision and reliability, with increasing interoperability across a growing
+variety of data types, and at intensifying speed is leading to an unprecedented concentration of power
+that threatens private and public liberties.
+
+What's more, automation and the increasing computerisation of all aspects of our lives
+both increase the power of information and decrease the cost of a number of intrusive
+behaviours that would be more easily kept in check if the perpetrator had to be in the same room as
+the victim.
+
+These asymmetries of information and of automation create commanding asymmetries of power. It is not
+so much that <q>code is law</q>  but rather that code is judge, jury, and executioner.
+
+The Web is for everyone ([[?For-Everyone]]). It is <q>a platform that helps people and provides a
+net positive social benefit</q> ([[?ETHICAL-WEB]], [[?design-principles]]). One of the ways in which the
+Web serves people is by protecting them in the face of asymmetries of power and this includes
+establishing and enforcing rules to govern the power of data.
+
+The Web is a social and technical system made up of [=information flows=]. Because this document
+is specifically about [=privacy=] as it applies to the Web, it also focuses on privacy around
+information flows.
+
+<dfn data-lt="governance">Data governance</dfn> is the system of [=rules=] that regulate [=information flows=]. When
+[=people=] are involved in these [=information flows=], [=data governance=] determines how
+these rules constrain and distribute the power of information between different [=actors=].
+The <dfn>rules</dfn> describe the way in which different [=actors=] may, must,
+or must not produce or [=process=] flows of information from, to, or about other [=actors=]
+([[?GKC-Privacy]], [[?IAD]]).
```

Comment:
I have mixed feelings here. Looking at institutional analysis, we can have:
* Rules: A specified set of participants may/must/must not do a specific thing under specific conditions, or else.
* Norms: A specified set of participants may/must/must not do a specific thing under specific conditions.
* Strategies: A specified set of participants should do a specific thing under specific conditions.

What we're working on is all three. Things that the UA can enforce are rules (eg. "or else the cookie doesn't get shared", "or else there's an exception"). Things that the UA can't enforce but that we have clear opinions on are norms (eg. "if you share the email address used for login, you suck"). Norms also include things that we can't enforce yet but that we'd like to turn into rules because we can see that the norms don't work. And then there are strategies, which is to say that we know are often good practices, but we can't really tell you everything about when you need to be doing them (eg. a lot of de-identification or retention considerations).

Maybe we could surface this distinction, but I'm not sure that it would help more than hinder. So I'm using "rule" as the more generic word (the generic term of art is "institutional statement" but let's not?), the term that's often used in governance discussion, and that doesn't sound like it's about some vague ethics.

Reply:
It's all principles now.